recipes/flaightidl/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "flaightidl" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f19969361b2167fc349a8ddef97833006083ebc7a39eef69f4968d6c987f7f89
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - protobuf >=3.5.0,<4.0.0
```

Comment:
Sorry, just noticed that this doesn't look like it's needed at package build time, only run time:
```suggestion
```